README.md

```diff
@@ -53,7 +58,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+The likely memory requirement for this is O(1) because it compares only two tiles at any given time so the memory requirement stays the same regardless of n tiles.
```

Comment:
# 0

But where does it store the tiles it is comparing? Remember, we're talking about a parser here. It's a type 2 grammar, so it will need a transition table and a pushdown stack. The list of tiles is going to end up (at least partially) on that stack, so it's O(n).